rpcserver.go

```diff
@@ -4351,7 +4354,7 @@ func (r *rpcServer) SubscribeChannelEvents(req *lnrpc.ChannelEventSubscription,
 					},
 				}
 			case channelnotifier.OpenChannelEvent:
-				channel, err := createRPCOpenChannel(r, graph,
+				channel, err := createRPCOpenChannel(r,
```

Comment:
If we touch this line, can we fix the formatting of the multi-line function call? See https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#additional-style-constraints-on-top-of-gofmt

Reply:
corrected.